src/styles/card.css

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
```

Comment:
do you really need flex here? it's useless now, cause you're using margin between element and it's perfectly enough